docker-entrypoint.sh

```diff
@@ -105,6 +105,10 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 
 		echo
 		for f in /docker-entrypoint-initdb.d/*; do
+			if [ ! -f "$f" ]; then
+		        	echo "$0: ignoring is not a file $f"; echo
+		        	continue
```

Comment:
This is a mix of spaces and tabs -- should be just tabs. :wink:

Also, would you mind changing the output to the following? (so that it matches the following output lines better)
```bash
echo "$0: ignoring $f (not a file)"
```